argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -36,6 +36,15 @@ include::modules/gitops-creating-rolloutmanager-custom-resource.adoc[leveloffset
 // Deleting a RolloutManager custom resource
 include::modules/gitops-deleting-rolloutmanager-custom-resource.adoc[leveloffset=+1]
 
+// Adding OpenShift Routes as a traffic manager for Argo Rollouts
+include::modules/con_adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts.adoc[leveloffset=+1]
+
+// Installing the rollout-manager Operator
+// include::modules/proc_installing-the-rollout-manager-operator.adoc[leveloffset=+2]
```

Comment:
remove this lines as they are not needed.

Reply:
Fixed.